enhancing/enhancer.spec.js

```diff
@@ -1,2 +1,125 @@
-const enhancer = require('./enhancer.js');
+const enhancer = require("./enhancer.js");
 // test away!
+describe("enhancer test suite", () => {
+  describe("repair test suite", () => {
+    it("should repair the durability of the item to 100 if less than 100", () => {
+      // arrange
+      const item = { name: "broadsword", enhancement: "0", durability: 80 };
+      const expected = {
+        name: "broadsword",
+        enhancement: "0",
+        durability: 100
+      };
```

Comment:
Having some sort of function for making item objects might help with repetition; for example:
```javascript
const makeItemMaker = name => ({
  e: enhancement = "0", d: durability = 100 
}) => ({
  name,
  enhancement.
  durability
});

const makeBroadsword = makeItemMaker('broadsword');

const item = makeBroadsword({ d: 80 });
const expected = makeBroadsword();
```
(this could be up in the `describe` scope rather than the `it` scope so that you can use it in other tests)

Reply:
Read [these docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) if there's anything weird looking in terms of syntax here. This isn't necessarily the best approach here, I'm just wanting to show alternatives to get you thinking about your own alternatives!

Reply:
Makes sense, I liked copy and pasting it too much I guess.